apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/RowFilters.cs

```diff
@@ -0,0 +1,380 @@
+        /// </para>
+        /// </remarks>
+        /// <param name="regex">
+        /// The RE2 regex used to match column families. Must not be null.
+        /// </param>
+        /// <returns>The created row filter.</returns>
+        public static RowFilter FamilyNameRegex(string regex) =>
+            new RowFilter { FamilyNameRegexFilter = GaxPreconditions.CheckNotNull(regex, nameof(regex)) };
+
+        // TODO: Copy docs from Interleave.Filters, but only once the formatting issues are resolved.
+        /// <summary>
+        /// Creates a new <see cref="RowFilter"/> instance which applies several
+        /// RowFilters to the data in parallel and combines the results.
+        /// </summary>
+        /// <param name="filters">
+        /// The filters to apply in parallel. Must not be null, or contain null elements.
+        /// </param>
+        /// <returns>The created row filter.</returns>
+        public static RowFilter Interleave(params RowFilter[] filters) =>
+            new RowFilter
+            {
+                Interleave = new RowFilter.Types.Interleave
+                {
+                    Filters = { Utilities.ValidateCollection(filters, nameof(filters)) }
+                }
+            };
+
+        /// <summary>
+        /// Creates a new <see cref="RowFilter"/> instance which matches all cells,
+        /// regardless of input. Functionally equivalent to leaving `filter` unset,
+        /// but included for completeness.
+        /// </summary>
+        /// <returns>The created row filter.</returns>
+        public static RowFilter PassAllFilter() =>
+            new RowFilter { PassAllFilter = true };
+
+        /// <summary>
+        /// Creates a new <see cref="RowFilter"/> instance which matches only cells
+        /// from rows whose keys satisfy the given RE2 regex. In other words, passes
+        /// through the entire row when the key matches, and otherwise produces an
+        /// empty row.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// Note that, since row keys can contain arbitrary bytes, the `\C` escape
+        /// sequence must be used if a true wildcard is desired. The `.` character
+        /// will not match the new line character `\n`, which may be present in a
+        /// binary key.
+        /// </para>
+        /// <para>
+        /// Note that string is implicitly convertible to <see cref="BigtableByteString"/>, so <paramref name="regex"/> can
+        /// be specified using a string as well and its UTF-8 representations will be used.
+        /// </para>
+        /// </remarks>
+        /// <param name="regex">
+        /// The RE2 regex used to match row keys.
+        /// </param>
+        /// <returns>The created row filter.</returns>
+        public static RowFilter RowKeyRegex(BigtableByteString regex) =>
+            new RowFilter { RowKeyRegexFilter = regex };
+
+        // TODO: Seems like this is 0-1. Verify and document it.
```

Comment:
And validate input?

Reply:
Perhaps. Again, not sure if we should validate what isn't documented. I've updated the TODO.